packages/office-desk.yaml

```diff
@@ -65,42 +66,33 @@ sensor:
 
   - platform: uptime
     name: Uptime
-  
-  - platform: custom
-    lambda: |-
-      auto desk_height_sensor = new DeskHeightSensor(id(desk_uart));
-      App.register_component(desk_height_sensor);
-      return {desk_height_sensor};
-    sensors:
-      id: "desk_height"
-      name: Desk Height
-      unit_of_measurement: cm
-      accuracy_decimals: 1
-      icon: "mdi:counter"
-      state_class: "measurement"
-      on_value_range:
-      - below: ${min_height}
-        then:
-          - switch.turn_off: switch_down
-      - above: ${max_height}
-        then:
-          - switch.turn_off: switch_up
-      on_value:
-        then:
-          - cover.template.publish:
-              id: desk_cover
-              position: !lambda |-
-                  // The sensor outputs values from min_height (cm) to max_height (cm)
-                  // We need to translate this to 0 - 1 scale.
-                  float position = (float(x) - float(${min_height})) / (float(${max_height}) - float(${min_height}));
-                  return position;
-          - component.update: set_desk_height
+
+  - platform: loctekmotion_desk_height
+    id: "desk_height"
+    name: Desk Height
+    on_value_range:
```

Comment:
I haven't been able to test this yet, but is this just the number entity? It seems the normal `sensor` is not present anymore?

Reply:
This just replaces the previous "custom" sensor with a "loctekmotion_desk_height" sensor. Note it's in the "sensor" section.

Reply:
@jamesmyatt I don't understand why we need `on_value_range` on the sensor? I assume this is a mistake, right?

Reply:
Nevermind! Lgtm